tests/testthat/test-profile_emissions_upstream.R

```diff
@@ -26,7 +26,68 @@ test_that("irrelevant columns in `ecoinvent_inputs` aren't in the output", {
   expect_false(hasName(unnest_company(out), "new"))
 })
 
-test_that("the new API is equivalent to the old API except for extra columns", {
+# test_that("the new API is equivalent to the old API except for extra columns", {
+#   local_options(readr.show_col_types = FALSE)
+#
+#   companies <- read_csv(toy_emissions_profile_any_companies())
+#   co2 <- read_csv(toy_emissions_profile_upstream_products_ecoinvent())
+#   europages_companies <- read_csv(toy_europages_companies())
```

Comment:
@maurolepore Similarly this test could also be removed

Reply:
Sounds good. See my other related comment.